src/App.jsx

```diff
@@ -1,40 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const onTabSelected = (tab) => {
+    setSelectedTabId(tab.id);
+  };
```

Comment:
1. rename your function a little 
2. select a full tab
```suggestion
  const onTabSelect = (tab) => {
    setSelectedTab(tab);
  };
```